src/redux/books/books.js

```diff
@@ -1,8 +1,18 @@
 const ADD = 'bookstore/books/ADD';
 const REMOVE = 'bookstore/books/REMOVE';
 
-const initialState = [];
-let currentId = 0;
+const initialState = [{
+  id: 0,
+  title: 'Zero To One',
+  author: 'Peter Thiel',
+},
+{
+  id: 1,
+  title: 'Hit Refresh',
+  author: 'Satya Nadella',
+},
+];
+let currentId = 2;
```

Comment:
- [ ] **[OPTIONAL]** I suggest you make use of the `uuidv4` library to generate unique ids for your books. It is more authentic and less error-prone. Check this [link](https://www.codegrepper.com/code-examples/javascript/how+to+use+uuidv4+in+react) on the example code of how to use it in React.